### ISSUE TITLE
[QUESTION] bitbucket backend and invite only

### ISSUE THREAD

**Issue by julienchazal** [netlify/netlify-cms#2327]
Hello
Is it possible to have "invite only" working with bitbucket backend to access admin please ?
I would like visitors to access admin after being invited and register only with mail/password (no authentication provider)
i did not manage to get it working
Thx

**Comment by tomrutgers** [comment]
Currently it's only possible tp login to the CMS with username and password when you use Git Gateway. Due to technical reasons, Git Gateway is not supported for the Bitbucket backend, so it's only possible to login with Bitbucket credentials. More information about authentication and backend can be found here: https://www.netlifycms.org/docs/authentication-backends/

TLDR; You can find more (shorter) info on logging in here as well: https://github.com/netlify/netlify-cms/issues/1340#issuecomment-446270634

**Comment by Pixelatex** [comment]
So if our repositories are hosted on bitbucket, there is no way a user can log in to the admin section without having a connected bitbucket account??

This is a major roadblock in using netlify-cms really :/ Is there no workaround for this?

**Comment by Pixelatex** [comment]
@tomrutgers so, If I got this right, the only way to use the netlify user system is if bitbucket gets integrated in the git gateway?

Is there any information on how/when/who this might be possible?

**Comment by erquhart** [comment]
There are API compatibility issues barring our support of Bitbucket. There's no timeline currently, as the issues are on Bitbucket's side.
